src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx

```diff
@@ -48,8 +52,18 @@ export const RemoveLiquidityModal = () => {
   const { publicKeyHash } = useSelectedAccountSelector();
   const styles = useRemoveLiquidityModalStyles();
   const assetsList = useAssetsListSelector();
-
-  const searchLp = assetsList.find(token => getTokenSlug(token) === LIQUIDITY_BAKING_LP_SLUG);
+  const notAccountedAssetsList = useTokensMetadataSelector();
+
+  let isPresentInMetadata;
+  if (LIQUIDITY_BAKING_LP_SLUG in notAccountedAssetsList) {
+    isPresentInMetadata = {
+      ...notAccountedAssetsList[LIQUIDITY_BAKING_LP_SLUG],
+      isVisible: true,
+      balance: '0'
+    };
+  }
+
+  const searchLp = assetsList.find(token => getTokenSlug(token) === LIQUIDITY_BAKING_LP_SLUG) ?? isPresentInMetadata;
```

Comment:
try this
```
const assetsList = useAssetsListSelector();
  const tokensMetadataRecord = useTokensMetadataSelector();

  const lpToken: TokenInterface = {
    ...emptyToken,
    ...tokensMetadataRecord[LIQUIDITY_BAKING_LP_SLUG],
    ...assetsList.find(token => getTokenSlug(token) === LIQUIDITY_BAKING_LP_SLUG)
  };
  ```
  if `lpToken.address === emptyToken.address` - load metadata from API